addons/mail/static/src/owl/components/composer_text_input/composer_text_input.xml

```diff
@@ -1,9 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
+    <t t-name="owl.UserStatus" owl="1">
```

Comment:
owl is not a good prefix, and if it's called in only one place I would just inline it.

And if it has to be separate, we should consider making it a component.